test/validation/shared.go

```diff
@@ -169,3 +172,24 @@ func ValidateRPResources(ctx context.Context, t *testing.T, expected *RPResource
 		}
 	}
 }
+
+// DoesCredentialExist checks if the credential is registered in the workspace and returns a boolean value.
+func DoesCredentialExist(t *testing.T, credential string) bool {
```

Comment:
Please keep the name consistent.

```suggestion
// CredentialExists checks if the credential is registered in the workspace and returns a boolean value.
func CredentialExists(t *testing.T, credential string) bool {
```